src/components/button/button.test.js

```diff
@@ -210,4 +193,216 @@ context("Test for Button component", () => {
+
+      cy.checkAccessibility();
+    });
+
+    it("should pass accessibility tests for DashedButtonIconBefore", () => {
+      CypressMountWithProviders(<stories.DashedButtonIconBefore />);
+
+      cy.checkAccessibility();
+    });
+
+    it("should pass accessibility tests for DashedButtonDisabled", () => {
+      CypressMountWithProviders(<storiesDefault.DashedButtonDisabled />);
+
+      cy.checkAccessibility();
+    });
+
+    it("should pass accessibility tests for DashedButtonFullWidth", () => {
+      CypressMountWithProviders(<storiesDefault.DashedButtonFullWidth />);
+
+      cy.checkAccessibility();
+    });
+
+    it("should pass accessibility tests for DashedButtonNoWrap", () => {
+      CypressMountWithProviders(<storiesDefault.DashedButtonNoWrap />);
+
+      cy.checkAccessibility();
+    });
+
+    it("should pass accessibility tests for DarkBackgroundButtonIconBefore", () => {
+      CypressMountWithProviders(<stories.DarkBackgroundButtonIconBefore />);
+
+      cy.checkAccessibility();
+    });
+
+    it("should pass accessibility tests for DarkBackgroundButtonIconAfter", () => {
+      CypressMountWithProviders(<stories.DarkBackgroundButtonIconAfter />);
+
+      cy.checkAccessibility();
+    });
+
+    it("should pass accessibility tests for DarkBackgroundButtonDisabled", () => {
+      CypressMountWithProviders(
+        <storiesDefault.DarkBackgroundButtonDisabled />
+      );
+
+      cy.checkAccessibility();
+    });
+
+    it("should pass accessibility tests for DarkBackgroundButtonFullWidth", () => {
+      CypressMountWithProviders(
+        <storiesDefault.DarkBackgroundButtonFullWidth />
+      );
+
+      cy.checkAccessibility();
+    });
+
+    it("should pass accessibility tests for DarkBackgroundButtonNoWrap", () => {
+      CypressMountWithProviders(<storiesDefault.DarkBackgroundButtonNoWrap />);
+
+      cy.checkAccessibility();
+    });
+
+    it("should pass accessibility tests for TertiaryButtonIconBefore", () => {
+      CypressMountWithProviders(<stories.TertiaryButtonIconBefore />);
+
+      cy.checkAccessibility();
+    });
+
+    it("should pass accessibility tests for TertiaryButtonIconAfter", () => {
+      CypressMountWithProviders(<stories.TertiaryButtonIconAfter />);
+
+      cy.checkAccessibility();
+    });
+
+    it("should pass accessibility tests for TertiaryButtonDestructive", () => {
+      CypressMountWithProviders(<storiesDefault.TertiaryButtonDestructive />);
+
+      cy.checkAccessibility();
+    });
+
+    it("should pass accessibility tests for TertiaryButtonDisabled", () => {
+      CypressMountWithProviders(<storiesDefault.TertiaryButtonDisabled />);
+
+      cy.checkAccessibility();
+    });
+
+    it("should pass accessibility tests for TertiaryButtonFullWitdth", () => {
+      CypressMountWithProviders(<storiesDefault.TertiaryButtonFullWitdth />);
+
+      cy.checkAccessibility();
+    });
+
+    it("should pass accessibility tests for TertiaryButtonFullWitdth", () => {
```

Comment:
Should read `TertiaryButtonNoWrap`.

Reply:
just to do not re-run chromatic again will change it in another PR